scipy/stats/tests/test_distributions.py

```diff
@@ -1380,27 +1380,32 @@ def test_sf_isf(self):
         
 class TestGammaGompertz(object):
     def test_pdf(self):
-        vals = gamma_gompertz.pdf(np.arange(0., 5., 1.), 1., 1.)
-        expected = [1., 0.367879441, 0.135335283, 0.0497870684, 0.0183156389]
+        vals = stats.gamma_gompertz.pdf(np.log(np.arange(1., 6., 1.)), 1., 2.)
+        expected = [0.5, 4./9., 0.375, 8./25., 5./18.]
```

Comment:
Unimportant note for future: `4./9. == 4/9` --- `/` division is floating point division also on python2 in scipy source files.

Reply:
Thank you for pointing out that division is float point in both python 2 and python 3 in scipy.